compiler/ballerina-lang/src/main/resources/compiler.properties

```
@@ -101,6 +101,9 @@ warning.checked.expr.invalid.usage.no.error.type.rhs=\
 warning.invalid.metadata.on.duplicate.enum.member=\
     cannot specify metadata on more than one duplicate enum member
 
+warning.strand.annotation.will.be.deprecated=\
+    usage of ''strand'' annotation will be deprecated
```

Missed this earlier. But this should be just
```suggestion
warning.strand.annotation.will.be.deprecated=\
    the ''strand'' annotation will be deprecated
```

right?

We can probably fix this separately.